auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -14,23 +14,34 @@ class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
     @tools.ormcache(skiparg=0)
-    def get_session_parameters(self, db):
+    def _auth_timeout_get_parameter_delay(self, db):
```

Comment:
I was trying to be granular here; with the new structure each param is required in different places, so I split `get_session_parameters` in two.

Alternatively I could have just called `get_session_parameters` in all the now required places, using the parameter returned that I need and discarding the other; however I was feeling the need to be frugal given that from what I can see these methods are running in a fairly hot code-path.

The other thing that occurs to me now is that this code now opens more cursors than before.
Technically this new code would open and close a cursor for each parameter read, whereas they were both read in a single cursor before.

Am I barking up the wrong tree here?
Should I be being less frugal and trusting the ORM layer to mitigate any repeat work ?